core/src/main/java/brooklyn/management/classloading/BrooklynClassLoadingContextSequential.java

```diff
@@ -101,6 +102,18 @@ public URL getResource(String resourceInThatDir) {
     }
 
     @Override
+    public Iterable<URL> getResources(String name) {
+        List<Iterable<URL>> resources = Lists.newArrayList();
+        for (BrooklynClassLoadingContext target : primaries) {
+            resources.add(target.getResources(name));
+        }
+        for (BrooklynClassLoadingContext target : secondaries) {
+            resources.add(target.getResources(name));
+        }
+        return Iterables.concat(resources);
```

Comment:
It's often the case that the application class loader will be contained multiple times in the BCLCs, even in a single one, leading resources.